doc/lua_api.txt

```diff
@@ -4117,6 +4248,14 @@ The Biome API is still in an experimental phase and subject to change.
     --  ^ Uses texture (string)
         playername = "singleplayer"
     --  ^ optional, if specified spawns particle only on the player's client
+        material_type_param = 0,
+    --  ^ optional, if specified spawns particle with
+    --  specified material type param, refer to "Particle blend" section
+        animation = {Animation definition},
+    --  ^ optional, note that particle and particle spawners have differences
+        glow = 0,
```

Comment:
@paramat @sfan5 Obviously fire should have glow level 15. If we use fixed value 4-8 independed from environment light level and emit this particle in full bright area, this particle will be visually darker, than same particle in dark area. Which is quite strange behaviour for material.

So, paramat, should we consider your comment as a feature request?